ledger/participant-state-metrics/src/main/scala/com/daml/ledger/participant/state/index/v2/metrics/TimedIndexService.scala

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) 2020 The DAML Authors. All rights reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package com.digitalasset.platform.state
+package com.daml.ledger.participant.state.index.v2.metrics
```

Comment:
Index service is specific to sandbox. please keep everything related to it in sandbox. basically `participant-state-index` is mostly not needed anymore, because there is now only a single daml-on-x stack.

Reply:
It only pulls it into _kvutils/app_, which depends on _sandbox_ anyway, but I agree, we should move it to _com.digitalasset.platform.apiserver_ in _sandbox_.

Reply:
It only pulls it into _kvutils/app_, which depends on _sandbox_ anyway, but I agree, we should move it to _com.digitalasset.platform.apiserver_ in _sandbox_.